web/search.dart

```diff
@@ -160,6 +162,7 @@ void initializeSearch(
     }
   });
 
+  var body = document.querySelector('body');
```

Comment:
Let's declare this variable as close as possible to where it is used, around line 450.

We can also null-assert (`!`) when we declare it since it is null-asserted on line 457.